soapbox/dotnet/README.md

```diff
@@ -0,0 +1,3245 @@
+# Clean Code concepts adapted for .NET/.NET Core
+
+If you liked `clean-code-dotnet` project or if it helped you, please give a star :star: for this repository. That will not only help strengthen our .NET community but also improve skills about the clean code for .NET developers in around the world. Thank you very much :+1:
```

Comment:
Is this original content or taken from somewhere else?

Reply:
@Vouzamo bump on this.  if this is content you wrote, please structure it using your name and add it to the index like the others. also we're moving the repo here, so if you can resubmit there that would be awesome, thank you so much! https://github.com/deptagency/engineering-soapboxes